test/passing/invalid_docstring.ml

```diff
@@ -0,0 +1 @@
+(**{v*)
```

Comment:
Is this going to display a warning if a fixed version of odoc is used?

Reply:
I forgot your comment about removing the test, I'll do that